vllm/config/parallel.py

```diff
@@ -482,12 +486,10 @@ def compute_hash(self):
             "_api_process_rank",
         }
 
-        from vllm.config.utils import get_hash_factors, hash_factors
-
-        factors = get_hash_factors(self, ignored_factors)
+        factors = get_compile_factors(self, ignored_factors)
         # Explicitly include backend affecting env factor as before
         factors["VLLM_ALL2ALL_BACKEND"] = str(envs.VLLM_ALL2ALL_BACKEND)
-        return hash_factors(factors)
+        return factors or {}
```

Comment:
`factors` always be a `dict` anyway?

```suggestion
        return factors
```